app/jobs/create_pulp_disk_space_notifications.rb

```diff
@@ -6,4 +6,8 @@ class CreatePulpDiskSpaceNotifications < ApplicationJob
   def perform
     Katello::UINotifications::Pulp::ProxyDiskSpace.deliver!
   end
+
+  def humanized_name
+    'Create Pulp disk space notifications'
```

Comment:
What about "Pulp disk space notifications"

Reply:
shouldn't these be I18n'd?

Reply:
Indeed